src/pages/develop/services.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Services
+---
+
+<Image src="https://res.cloudinary.com/railway/image/upload/v1645141323/docs/servicesgif.gif"
+alt="Gif of the Services view"
+layout="intrinsic"
+width={800} height={518} quality={100} />
+
+A Railway service is a deployment target for your application. There are four types of services.
+
+- GitHub Services
+- Database Services
+- Starter Services
+- Empty Services
```

Comment:
suggestion: Reword this slightly. There are not really only 2 types of services (database and non-database). An empty service is just a GitHub service without a repo attached. A starter is a pre-configured group of other services.